agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -422,14 +494,15 @@ public void validate(ProcessorType processorType) throws FriendlyException {
                                 "Learn more about " + processorType + " processors here: https://go.microsoft.com/fwlink/?linkid=2151557");
             }
             for (String rule : rules) {
-                ProcessorConfig.isValidRegex(rule, processorType);
+                validateRegex(rule, processorType);
             }
         }
     }
 
     public static class ProcessorIncludeExclude {
         public MatchType matchType;
         public List<String> spanNames = new ArrayList<>();
+        public List<String> metricNames = new ArrayList<>();
```

Comment:
if not renaming to METRIC, can you rename this to metricFilterNames for consistency.